.github/workflows/run-tests.yml

```diff
@@ -0,0 +1,33 @@
+name: Run test
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    container:
+      image: 4141done/njs-test-runner:1.23.2_0.7.9
+      env:
+        # This is take from the `PATH` variable in the target docker container.  The HOME variable
+        # will be overridden even if set here.
+        PATH: /root/.asdf/shims:/root/.asdf/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
```

Comment:
**annotation**:
I'm having to do this because I want to have the local docker-based testing flow use the same container as the CI.  This makes the CI flow more portable and it also uses the local `.tool-versions` file so we can be sure of the same nodejs versions between dev and CI.